radar/platforms-and-services/azure-devops.md

```diff
@@ -0,0 +1,17 @@
+---
+title: "Azure DevOps"
+ring: hold
+quadrant: "platforms-and-aoe-services"
+tags: [cloud, pipelines, cicd]
+---
+
+[Azure DevOps](https://azure.microsoft.com/it-it/services/devops/) is a set of
+services offered by Microsoft for the development of software in the cloud. It
+is a complete solution that covers the entire software development lifecycle and
+allows you to plan, develop, test, deploy and monitor applications with a single
+tool.
+
+Since we switched to GitHub Actions, that offers a similar set of features, but
```

Comment:
I agree that Azure DevOps now is not the preferred CI/CD tool but we still need it for scheduled pipeline and PCI project.

https://github.com/marketplace/actions/keepalive-workflow and similar actions requires:
1. a github bot user with admin permission on repository if branch protection enabled
2. branch protection is not enabled (https://github.com/gautamkrishnar/keepalive-workflow/blob/master/library.js#L21)

We also use Azure DevOps in PCI environments where repos are private and we need
1. strict deploy approve process
2. audit logs integration with Azure Sentinel (now with GitHub Enterprise this integration should be supported, but not yet implemented)

So `hold` ring I don't know if is the most appropriate.

Reply:
Can we say hold for non PCI environments and adopt for PCI environments?

Reply:
What do you think in adopting [this new GitHub Service](https://github.blog/2023-06-05-announcing-the-stale-repos-action/) to identify inactive repos? Let's say to identify inactive repos for 85 days (auto archive is after 90d) and automatically creates/update/wathever an issue/a setting or whatsover.

Do you know any blocker this approach would have that I am not aware of?

Reply:
we also use Azure Devops for the creation of certificates and for all IAC projects in the payments area.

We can say, that it is used for projects, pci mandatory for now, and as an exception to be checked every time

Reply:
Ok, let's close this one then.